mautic/mautic
# [UX] Shortcuts for campaign-related resources

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR is based on other PRs. **We need them merged before this one becomes available for code review.**

Dependencies:
- #13735 
- #13772
- #13848 
- #13881 

---

This pull request adds shortcuts on the campaign page for users to create campaign-related resources directly.
It can be confusing for new users to understand that they have to use specific email types and forms to be able to use them in campaign workflows. The idea here is to create a visible relationship between all of them.


This way, we:
- reduce the learning curve for new users
- improve specific workflows 
- enhance user experience 

See these clickable tiles:

![image](https://github.com/mautic/mautic/assets/116097999/77a7d983-4a36-474c-a710-2a6b6c9a2954)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Campaigns
3. Click on each tile and create each resource to make sure they're the right type at the end, when looking at the list tables
